.github/workflows/release.yml

```diff
@@ -52,6 +52,7 @@ jobs:
     environment: release
```

Comment:
Note, we already run this in the `release` environment, so I'm updating the npm OIDC settings to require that environment too.